src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final FigureEnum DEFAULT_FIGURE_NAME = FigureEnum.CIRCLE;
+    public static final double DEFAULT_CIRCLE_RADIUS = 10;
+    public static final String DEFAULT_CIRCLE_COLOR = Color.WHITE.name();
+    public static final int MAX_FIGURES = 6;
```

Comment:
Bad name. `MAX_FIGURES` means the maximum number of figures, not the maximum size or sth like this